components/d2l-filter-dropdown/d2l-filter-dropdown-category.js

```diff
@@ -0,0 +1,163 @@
+import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
+import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
+import { afterNextRender } from '@polymer/polymer/lib/utils/render-status.js';
+import 'd2l-tabs/d2l-tab-panel-behavior.js';
+import 'd2l-inputs/d2l-input-search.js';
+import 'd2l-menu/d2l-menu.js';
+import 'd2l-colors/d2l-colors.js';
+import './d2l-filter-dropdown-localize-behavior.js';
+
+/**
+ * @customElement
+ * @polymer
+ */
+
+class D2LFilterDropdownCategory extends mixinBehaviors([D2L.PolymerBehaviors.Tabs.TabPanelBehavior, D2L.PolymerBehaviors.FilterDropdown.LocalizeBehavior], PolymerElement) {
+	static get template() {
+		return html`
+			<style include="d2l-tab-panel-styles">
+				:host {
+					margin: -1px -1rem 0 -1rem;
+					border-top: 1px solid var(--d2l-color-gypsum);
+					padding: 1.2rem 1rem 0 1rem;
```

Comment:
Looks like some margins and padding that will need to be reversed in RTL.

Reply:
The left and right are the same for both, it's just the top and bottom that needed to be customized